app/models/provider.rb

```diff
@@ -45,8 +45,10 @@ class Provider < ApplicationRecord
 
   scope :changed_since, ->(datetime) do
     if datetime.present?
-      where("updated_at >= ?", datetime).order(:updated_at, :id)
-    end
+      where("last_published_at >= ?", datetime)
+    else
+      where("last_published_at is not null")
```

Comment:
This feels odd but I assume you have a valid reason it.

Reply:
the `is not null` is there to prevent unpublished providers escaping into the ucas system prematurely, there's probably a nicer syntax or more DRY way of doing this, any ideas?

Reply:
went round the houses trying to optimise this. do later.